docs/pages/includes/kubernetes-access/rbac.mdx

```diff
@@ -28,54 +29,63 @@ spec:
   deny: {}
 ```
 
-Retrieve your user:
+3. Retrieve your Teleport user:
 
 ```code
 $ TELEPORT_USER=myuser
 $ tctl get user/${TELEPORT_USER?} > user.yaml
 ```
 
-Add `kube-user` to your Teleport user's list of roles:
+4. Add `kube-access` to your Teleport user's list of roles:
 
 ```diff
    roles:
-   - access
-   - auditor
-+  - kube-user
++  - kube-access
 ```
 
-Apply your changes:
+5. Apply your changes:
 
 ```code
-$ tctl create -f kube-user.yaml
+$ tctl create -f kube-access.yaml
 $ tctl create -f user.yaml
 ```
 
-In order for the `viewers` group to have access to anything, it must be granted permissions in your Kubernetes cluster.
-Create a file called `viewers-bind.yaml` with the following contents:
+Now that Teleport RBAC is configured, you can **authenticate** to your Kubernetes cluster, but authentication
+alone does not allow you to actually do anything in the Kubernetes cluster; for that you need **authorization**.
+
+### Kubernetes Authorization
+
+To configure **authorization** within your Kubernetes cluster, you need to create Kubernetes RoleBindings or
+ClusterRoleBindings which grant permissions to the subjects listed in `kubernetes_users` and
+`kubernetes_groups`.
+
+For example, you can grant some limited read-only permissions to the `viewers` group used in the `kube-access`
+role defined above:
+
+1. Create a file called `viewers-bind.yaml` with the following contents:
```

Comment:
As with my earlier comment, I don't think the numbers help here, since this is passage includes body text and code snippets of varying length.